src/open_inwoner/utils/time.py

```diff
@@ -38,3 +38,7 @@ def has_new_elements(
     otherwise
     """
     return any(is_new(elem, attribute_name, delta) for elem in collection)
+
+
+def convert_datetime_to_iso_8601_with_z_suffix(dt: dt.datetime) -> str:
+    return dt.strftime("%Y-%m-%dT%H:%M:%SZ")
```

Comment:
Oops, it doesn't actually say anything about the datetimes being UTC, so I suppose I'll just format them the same way as the API (e.g. `"2012-08-13 16:13:07"`)

Reply:
Annoyingly the dates in the Laposta API docs are not strict ISO datetimes because they have a space instead of a 'T' to separate the date and time components. It still parses in Python `datetime.fromisoformat` because Python has some exceptions (https://docs.python.org/3/library/datetime.html#datetime.datetime.fromisoformat):

> The T separator may be replaced by any single unicode character.

Reply:
I guess this means they are timezone naive but local Dutch so let's not touch them.